nanoFramework.Json/Converters/DateTimeConverter.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Text;
+
+namespace nanoFramework.Json.Converters
+{
+    internal sealed class DateTimeConverter : IConverter
+    {
+        /// <summary>
+        /// <inheritdoc/>
+        /// </summary>
+        public string ToJson(object value)
+        {
+            return "\"" + DateTimeExtensions.ToIso8601((DateTime)value) + "\"";
+        }
+
+        /// <summary>
+        /// <inheritdoc/>
+        /// </summary>
+        public object ToType(object value)
+        {
+            // Not sure hwo this one should work
```

Comment:
well, when is this one used? :-)

Reply:
Not sure if it's used at all. "ToType" method should be used when converting from json to object, but looks like DateTime never hits that. 
I left that not implemented yet all tests are passing... It's something for next PR, this one is already too big...

Reply:
The same situation with bool converter.

Reply:
OK. Good like this then.